tests/Resources/Page/LoginPage.robot

```diff
@@ -3,9 +3,9 @@ Library  Selenium2Library
 
 
 *** Variables ***
-${UI} =  https://console-openshift-console.apps.modh-qe.dev.datahub.redhat.com/
-${KUBEADMIN} =  kubeadmin
-${KUBEPASSWD} =  DYYw5-KSmW7-F9gQt-gzJpV
+${UI} =  %{CONSOLE_URL}
+${KUBEADMIN} =  %{KUBEADMIN}
```

Comment:
I kept this as is but the script wont start until the ENV variables are setup properly,